contents/teams/max-ai/index.mdx

```diff
@@ -5,3 +5,30 @@ showTitle: true
 hideAnchor: false
 template: team
 ---
+
+Max AI lets you use PostHog's products through a chat interface.
+
+## Working with other small teams:
+
+Creating AI products involves specialist knowledge, a bit like infrastructure.
+
+Our other small teams will always want to make their products work well with MaxAI's interface.
+
+To do this, we start engineers out in the MaxAI team. They will ship whatever the current roadmap is.
+
+When other teams need help, for example -
+
+* the warehouse team wants to build natural language -> SQL
+* the product analytics team wants to build natural language -> insights
+* the session replay team wants to build natural language -> find relevant recordings
+
+Then the MaxAI team will send one of their engineers out to the other small team for one or several sprints, pairing with someone on the other team.
+
+This naturally distributes AI knowledge through the organization _and_ means the AI features we ship are high quality and fit with our platform properly.
+
+Over time, this team should maintain a feature ownership table.
+
+ie:
+
+Michael -> product analytics + warehouse
+Georgiy -> session replay
```

Comment:
I'm not sure I agree with this. This type of functionality is becoming table stakes across the industry. It seems like most of our product engineers should learn how to work with this type of tooling. I think the goal for these temporary embeddings in other teams should be to help them get it set up but also help them learn the skills they need in order to maintain it almost completely by themselves.

Reply:
For sure, in this case think of the Max team as infra consultants. It definitely should not be "Georgiy gets told users need session replay analysis, implements it with AI magic, and then disappears from the replay product".

Rather: "Georgiy and [owner on the DW side] go through the user problem, brainstorm the AI experience to solve it, then [DW person] builds the thing using our AI primitives – Georgiy just ensures the UX works well with Max as a whole, and updates the primitives to fully support the use case".

Down the line this should be almost fully hands-off as we build out and document the Max platform, just for the first few use cases we need to work very closely

Reply:
Yeah that makes sense to me - it's this block at the end which I think doesn't convey that. I think the doc is great without it. 

```suggestion
```